seed/static/seed/js/controllers/inventory_detail_analyses_controller.js

```diff
@@ -103,6 +103,14 @@ angular.module('BE.seed.controller.inventory_detail_analyses', [])
         })
       }
 
+      $scope.inventory_display_name = function() {
+        let field = $scope.org.property_display_field;
+        if (!$scope.item_state[field]) {
+          field = 'address_line_1';
+        }
+        return $scope.item_state[field] ? $scope.item_state[field] : '(invalid display field)';
```

Comment:
I like what you've got here, very clean. I'm wondering if this message might confuse some folks who don't manage the org settings. What do you think about something like `(no ${field})`, or even better `(no ${display name for the field})`, kind of like SEED displayed before?
Also I'm wondering if there is a way we could help org managers fix this if they see this message, ie some help message or link to settings. Not sure if either option is good UI, but in general its useful to say how to fix things in addition to reporting a problem

Reply:
I was actually thinking about putting in something along those lines so I'm glad you brought it up too.  I'll update with something more clear and possibly a tooltip to direct users to fix it.

Reply:
<img width="613" alt="screencapture" src="https://user-images.githubusercontent.com/564231/106200189-da81b900-6173-11eb-8f96-cf05a717caaa.png">

Reply:
ohh looks very nice!